crates/router/src/connector/rapyd.rs

```diff
@@ -64,6 +64,10 @@ impl ConnectorCommon for Rapyd {
         "rapyd"
     }
 
+    fn get_currency_unit(&self) -> api::CurrencyUnit {
+        api::CurrencyUnit::Base
+    }
```

Comment:
As per Rapyd's Use Case the amount will be `Minor`

```suggestion
    fn get_currency_unit(&self) -> api::CurrencyUnit {
        api::CurrencyUnit::Minor
    }
```

Reply:
Sorry for that major fault. I don't know why I figured out that it should be base currency from the documentation. I'll do the corrections immediately.